src/configuration.ts

```diff
@@ -41,3 +41,26 @@ async function readYamlFile(filePath: string) {
     const fileBuffer = await fs.readFile(filePath);
     return YAML.parse(fileBuffer.toString('utf8')) as Configuration;
 }
+
+function mergeConfigurations(configurations: Configuration[]): Configuration {
```

Comment:
Maybe we should rethink about whether or not we want an administrator to be able to define multiple config files. At the very least, every error that we display regarding the configuration should also display which file(s) caused that error.

Reply:
Yes, lets consider this issue in a separate ticket.